### DISCUSSION TITLE
Android version

### DISCUSSION THREAD

**Discussion by ghost** [ahrm/sioyek#684] [General]
@ahrm Could you comment here on the progress of the android version?

I am very excited with this. In my case, I want an android version for pdf interoperability. 

My workflow sometimes requires me to send a pdf that is partially highlighted to my android tablet via [syncthing](https://syncthing.net/). So what I would need is to edit those highlights between any device and then to sync them.

**Comment by ahrm** [comment]
I would say the first 95% of the work is done. Now just the second 95% remains!

![-5803300566032562789_121](https://user-images.githubusercontent.com/6392321/230719787-1fa4b571-3c4c-42f7-8ace-4464db3795ee.jpg)
